Source/Core/DolphinWX/Input/HotkeyInputConfigDiag.cpp

```diff
@@ -68,6 +69,31 @@ HotkeyInputConfigDialog::HotkeyInputConfigDialog(wxWindow* const parent, InputCo
   // Frame advance is an example of a typical TAS tool.
   notebook->AddPage(tab_tas, _("TAS Tools"));
 
+  if (withDebugging)
+  {
+    // Debugging
+    auto* const tab_debugging = new wxPanel(notebook);
```

Comment:
> The only reason the comment is here is to follow what I did on the other tabs, it's just to get which tabs has which code.

Right, but 'comments as dividers' is generally not a good idea. If you put them into their own function, it makes the UI creation easier to follow, since the top level will just be:

```cpp
auto* const notebook = new wxNotebook(this, wxID_ANY);
notebook->AddPage(CreateGeneralTab(), _("General"));
...
notebook->AddPage(CreateDebuggerTab(), _("Debugging"));
```
etc

> As for the function, I am confused if that applies to just this one or the other tabs too because it's like any other tab, the only difference is the debug mode check.

If you prefer, I can do this in a different PR.